Makefile

```diff
@@ -448,6 +448,10 @@ ifneq ($(OS), WINNT)
 endif
 ifeq ($(OS), Linux)
 	-$(JULIAHOME)/contrib/fixup-libstdc++.sh $(DESTDIR)$(private_libdir)
+	# We need to bundle ca certs on linux now that we're using libgit2 with ssl
+ifneq ($(shell cat $(shell openssl version -d | cut -d '"' -f 2)/cert.pem),)
+	-cp $(shell openssl version -d | cut -d '"' -f 2)/cert.pem $(DESTDIR)$(datarootdir)/julia/
+endif
```

Comment:
Definitely MPL 2.0. The FSF has always maintained that compilers etc. do not affect the license of the code being transformed.